src/external/rollcall.ts

```diff
@@ -157,20 +153,18 @@ export default async (): Promise<RollCallClient> => {
   };
 
   const release = async (resovledIndex: Index): Promise<boolean> => {
-    logger.info(
-      `Requesting Rollcall to release index ${resovledIndex.indexName} to alias ${aliasName}`,
-    );
+    logger.info(`Requesting Rollcall to release index ${resovledIndex.indexName} to alias ${aliasName}`);
     const url = urljoin(`${rootUrl}`, `/aliases/release`);
 
     const req = await convertResolvedIndexToIndexReleaseRequest(resovledIndex);
 
-    const acknowledged = (await fetch(url, {
+    return await fetch(url, {
       method: 'POST',
       body: JSON.stringify(req),
       headers: { 'Content-Type': 'application/json' },
-    }).then(res => res.json())) as boolean;
-
-    return acknowledged;
+    })
+      .then(_ => true)
+      .catch(_ => false);
```

Comment:
nice

Reply:
hmmm... should log the returned failure.